vividus-tests/src/main/resources/story/system/mobile_app/MobileIOSDprCheck.story

```diff
@@ -0,0 +1,15 @@
+Scenario: Step verification: When I $actionType baseline with name `$name` for device with low dpr
```

Comment:
Can we reuse existing baselines, to avoid the creation of new ones? Less baselines = less support

Reply:
For an example:
1. Establish baseline using small dpr simulator
2. Compare against existing one https://github.com/vividus-framework/vividus/blob/master/vividus-tests/src/main/resources/baselines/ios-context.png

Reply:
I can use `ios-full-page.png` for low dpr

Reply:
story for iOS updated